releases/keycloak.yaml

```diff
@@ -75,8 +75,11 @@ releases:
           {{- if eq (env "KEYCLOAK_FORECASTLE_EXPOSE" | default "true") "true" }}
           forecastle.stakater.com/expose: "true"
           forecastle.stakater.com/appName: "Keycloak"
-          {{- if env "FORECASTLE_INSTANCE_NAME" }}
-          forecastle.stakater.com/instance: "{{ env "FORECASTLE_INSTANCE_NAME" }}"
+          {{- if coalesce (env "KEYCLOAK_FORECASTLE_INSTANCE_NAME") (env "FORECASTLE_INSTANCE_NAME") }}
+          forecastle.stakater.com/instance: '{{ coalesce (env "KEYCLOAK_FORECASTLE_INSTANCE_NAME") (env "FORECASTLE_INSTANCE_NAME") }}'
+          {{- end }}
+          {{- if env "KEYCLOAK_FORECASTLE_GROUP" }}
```

Comment:
@Nuru 
how do we set different `KEYCLOAK_FORECASTLE_GROUP` ENV var for diff sections in `keycloak-gatekeeper-services.yaml`?

Reply:
This `KEYCLOAK_FORECASTLE_GROUP` variable is for where in the portal the Keycloak Admin UI shows up. Gatekeeper services' groups can be set via `forecastleGroupName` as of `cloudposse/hemlfiles` release 0.55.1